blog/_posts/2025-08-14-node-autoprovisioning.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Announcement - Node Auto Provisioning"
+description: "Learn about Node Auto Provisioning for Azure Kubernetes Service, based on Karpenter, and how it can simplify the scaling experience for your workloads on AKS."
+date: 2025-08-14
+author: Wilson Darko # must match the authors.yml in the _data folder
+categories: 
+- general, add-ons, compute
+---
+
+We’re excited to announce the General Availability (GA) of Node Auto Provisioning (NAP) for Azure Kubernetes Service (AKS)!
+
+Managing dynamic workloads in Kubernetes often leads to overprovisioning, idle resources, and operational overhead from maintaining pre-configured node pools. [Karpenter](https://karpenter.sh/), the open-source CNCF project, allows for compute-optimized node provisioning, that offers flexibility, cost savings, and simplicity.
+
+With node auto provisioning (NAP), our managed add-on for the open-source [Karpenter](https://karpenter.sh/) project on AKS, AKS now automatically provisions single-instance nodes (VMs) in response to pending pod requirements, and optimizes the scaling experience by provisioning right-sized compute in the more efficient and cost-effective manner. With NAP, we bring a new approach to autoscaling and provisioning that outperforms cluster autoscaler in compute efficiency, cost, and ease of use. NAP offers greater flexibility, allowing many SKU sizes, spot and on-demand capacity, and multiple architecture types such as AMD and ARMx64 in the same cluster, without the need for separate node pools.
+
+![nap-ga-announcement](/assets/images/aks-nap/nap-ga-announcement.jpg)
+
+## What’s New with NAP since Preview Launch
+
```

Comment:
Perhaps worth deep diving into a section on 1 or 2 key important bits from the new announcement in your blog for example: Improved ugprade experience and Observability could be a deep dive topic ?